### DISCUSSION TITLE
Sesion 17/02 ¿?

### DISCUSSION THREAD

**Discussion by Eholguin05** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#13] [Q&A] [answered]
Buenas tardes, sabran si hoy habra sesion en vivo?,... segun el video de Rodrigo comenta que son los lunes, miercoles y Jueves a las 19hrs.

gracias!!

**Comment by LorenzoShelley** [comment]
Sí, hoy hay live

**Comment by BrawnyGG** [comment, marked as answer]
Si hay live, te dejo link...
https://teams.microsoft.com/l/meetup-join/19%3ameeting_YjE4OGIyMjEtZTc0ZC00ZGE5LTk5OGUtMjkxMDE1NTQ4OTg5%40thread.v2/0?context=%7b%22Tid%22%3a%224ae54b05-b77e-4224-aef1-8661422e0816%22%2c%22Oid%22%3a%2296eeb66f-b984-4083-8ea6-74c34a07a34a%22%2c%22IsBroadcastMeeting%22%3atrue%7d

**Reply by Eholguin05** [reply]
MUCHAS GRACIAS!!!, perdon si la pregunta es boba, de donde sacaron el lnk, no pude ser capaz de visualizarlo en teams.

**Reply by BrawnyGG** [reply]
Ninguna pregunta es tonta jeje

**Tienes que irte a la pestaña "Teams" y posteriormente darle click a la Misión Fronted - JS:**
![image](https://user-images.githubusercontent.com/98553492/154598885-98bec2ef-7fca-45f6-9f1a-87bad457dab5.png)

**Una vez ya dentro del equipo, te direccionará por defecto al canal "General", tienes que darle click al canal "Semana 1", ahí han estado subiendo todas las sesiones en vivo :)** 
![image](https://user-images.githubusercontent.com/98553492/154599092-6ee6cf25-7284-4f25-9d0f-c0c03f21bd90.png)

Espero haberte ayudado!

**Reply by Eholguin05** [reply]
ohhhhhhhhhh muchas gracias, es que aun tenia abierto el team de onBoarding. Saludos y grcias!
